src/oxoPatterns/oxoPattern2.java

```diff
@@ -37,9 +37,8 @@ public static void main(String[] args) {
                 System.out.println();
 
             }
-        }catch (Exception e){
+        } catch (NumberFormatException IllegalArgumentException) {
```

Comment:
why is there different exception name and type? Also i think there will be InputMissmatchException.